app/_how-tos/generate-a-collection-from-an-api-specification.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Generate a collection from an API specification
+
+products:
+    - insomnia
+
+tags:
+  - collections
+  - documents
+
+tldr: 
+  q: How do I generate a collection from an API spec?
+  a: Create or import an API spec and define environment variables, then click cog icon and click Generate collection.
+
+prereqs:
+  inline:
+    - title: Create an API specification
+      content: |
+        You can either [create an empty document]() and design your specification from scratch, or [import an existing specification](). This example uses the [Konnect API Products specification](https://docs.konghq.com/konnect/api/api-products/latest/).
```

Comment:
I think this could be a great opportunity to use [Kong Air](https://github.com/Kong/KongAir/blob/main/sales/bookings/package.json) for tutorials. We could use one of the specs in that repo for a prereq

Reply:
I wonder if we can provide the example spec contents in line so they can just copy/paste from this page. When I was doing this, I was a bit confused about where the spec went in Insomnia because I didn't realize I could copy/paste it right into the body, so I might say something along the lines of "Paste the spec directly into your document in Insomnia or import it in by clicking the **Import OpenAPI** drop down menu."

Reply:
I like the idea of using Kong Air, but when I import the specs in Insomnia I get a bunch of errors and warnings (which seem to be specific to Insomnia, because Swagger Editor doesn't report any errors on the same specs)